packages/Card/src/components/Avatar/Avatar.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
```

Comment:
Maybe a question for @nahumzs shouldn't Avatar be it's own component?

Reply:
I think mentioned in the PR description that for now it will live here. extracting it laters

Reply:
Can we put a TODO: Comment here to extract it as its own package later?